radicle-node/src/test/handle.rs

```diff
@@ -21,6 +21,7 @@ impl radicle::node::Handle for Handle {
     type Error = HandleError;
     type Sessions = service::Sessions;
     type FetchResult = FetchResult;
+    type Seeds = Seeds;
```

Comment:
Any reason this is an associated type and not fixed for all implementors? I think we're using the same concrete type everywhere?

Reply:
Ya, I initially thought it was the right approach because `FetchResult` was also parameterised. But to be honest, this whole `Handle` thing is a bit confusing for me. I've asked some questions there https://github.com/radicle-dev/heartwood/issues/271#issuecomment-1440484944.

I can change this to just be `Vec<Seed>` in the function, but would appreciate clearing up my questions in the other issue :blush: